crates/curp/src/members.rs

```diff
@@ -138,16 +138,26 @@ impl ClusterInfo {
     pub fn from_cluster(
         cluster: FetchClusterResponse,
         self_peer_urls: &[String],
+        self_client_urls: &[String],
         self_name: &str,
     ) -> Self {
         let mut member_id = 0;
+        let sorted_self_addr = self_peer_urls.iter().cloned().sorted().collect::<Vec<_>>();
```

Comment:
We can compare two iter directly without collecting them into Vec.

Reply:
Done